backend/app/config.py

```
@@ -0,0 +1,33 @@
+
+    # CORS設定
+    cors_origins: list[str] = ["http://localhost:3000"]
+
+    # ログ設定
+    log_level: str = "INFO"
+
+    # データベース設定
+    database_url: str = "sqlite+aiosqlite:////home/tsk/sync/git/claude-work/data/claudework.db"
+
+    # 認証設定
+    auth_token: str = "development_token_change_in_production"
```

Ruff (0.14.8)

Possible hardcoded password assigned to: "auth_token"

(S105)